test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -69,7 +69,7 @@ def run_smdebug_test(
         LOGGER.error(f"Caught exception while trying to run test via fabric. Output: {debug_output.stdout}")
         raise
 
-    # LOGGER.info(test_output.stdout) # Uncomment this line for a complete log dump
+    LOGGER.info(test_output.stdout)  # Uncomment this line for a complete log dump
```

Comment:
are we keeping this uncommented?

Reply:
I had uncommented this to help debug the initial failures that were fixed in #577 .
Will revert.